### ISSUE TITLE
Parsing error: Expecting Unicode escape sequence \uXXXX

### ISSUE THREAD

**Issue by StigVanbrabant** [eslint/eslint#10069]
**Tell us about your environment**

* **ESLint Version:**
4.15.0
* **Node Version:**
8.3.0
* **npm Version:**
5.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
// https://eslint.org/docs/user-guide/configuring

module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint'
  },
  env: {
    browser: true,
  },
  // https://github.com/vuejs/eslint-plugin-vue#priority-a-essential-error-prevention
  // consider switching to `plugin:vue/strongly-recommended` or `plugin:vue/recommended` for stricter rules.
  extends: ['plugin:vue/essential', 'airbnb-base'],
  // required to lint *.vue files
  plugins: [
    'vue',
    'html'
  ],
  // check if imports actually resolve
  settings: {
    'import/resolver': {
      webpack: {
        config: 'build/webpack.base.conf.js'
      }
    },
    'html/indent': '+2',
    'html/report-bad-indent': 'error',
  },
  // add your custom rules here
  rules: {
    // don't require .vue extension when importing
    'import/extensions': ['error', 'always', {
      js: 'never',
      vue: 'never'
    }],
    // disallow reassignment of function parameters
    // disallow parameter object manipulation except for specific exclusions
    'no-param-reassign': ['error', {
      props: true,
      ignorePropertyModificationsFor: [
        'state', // for vuex state
        'acc', // for reduce accumulators
        'e' // for e.returnvalue
      ]
    }],
    // allow optionalDependencies
    'import/no-extraneous-dependencies': ['error', {
      optionalDependencies: ['test/unit/index.js']
    }],
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-shadow': 0,
    'no-underscore-dangle': 0,
    'no-param-reassign': 0,
    'no-restricted-syntax': 0,
    'linebreak-style': 0,
    'o-absolute-path': 0,
    'no-new': 0,
    'no-console': 0,
    'no-bitwise': 0,
    'no-useless-escape': 0,
    'no-octal-escape': 0,
    'global-require': 0,
    'import/no-unresolved': 0,
    'no-param-reassign': 0,
    'valid-jsdoc': ['error', {
      'requireReturn': true,
      'requireReturnType': true,
      'requireParamDescription': true,
      'requireReturnDescription': true
    }],
    'prefer-destructuring': ['error', {
      'VariableDeclarator': {
        'array': false,
        'object': false
      },
      'AssignmentExpression': {
        'array': false,
        'object': false
      }
    }, {
      'enforceForRenamedProperties': false
    }],
    'require-jsdoc': ['error', {
        'require': {
            'FunctionDeclaration': true,
            'MethodDefinition': true,
            'ClassDeclaration': true
        }
    }],
    'max-len': ['error', {
      'code': 150
    }]
  }
}

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
const script = '<script type="text/javascript" src="..."><\/script>\n<noscript><img src="..." width="1" height="1"></noscript>';
```

```bash
webpack
```

**What did you expect to happen?**

For the linter to succeed because it all works when i turn of eslint.

**What actually happened? Please include the actual, raw output from ESLint.**

 ✘  http://eslint.org/docs/rules/  Parsing error: Expecting Unicode escape sequence \uXXXX

**Comment by not-an-aardvark** [comment]
Hi, thanks for the report. However, I can't seem to reproduce this issue in the [demo](https://eslint.org/demo/#eyJ0ZXh0IjoiY29uc3Qgc2NyaXB0ID0gJzxzY3JpcHQgdHlwZT1cInRleHQvamF2YXNjcmlwdFwiIHNyYz1cIi4uLlwiPjxcXC9zY3JpcHQ+XFxuPG5vc2NyaXB0PjxpbWcgc3JjPVwiLi4uXCIgd2lkdGg9XCIxXCIgaGVpZ2h0PVwiMVwiPjwvbm9zY3JpcHQ+JzsiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjgsInNvdXJjZVR5cGUiOiJtb2R1bGUiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnt9LCJlbnYiOnt9fX0=); the code lints fine without any errors.

**Comment by StigVanbrabant** [comment]
Yeah when i remove the use of Vue eslint plugin the error goes away, so the problem will be in there.

**Comment by not-an-aardvark** [comment]
Closing because this doesn't seem to be an issue with ESLint itself.
